admin-dev/themes/new-theme/js/pages/product/edit/product-form-model.ts

```diff
@@ -24,12 +24,19 @@
  */
 
 import BigNumber from 'bignumber.js';
+import EventEmitter from '@components/event-emitter';
 import ObjectFormMapper from '@components/form/form-object-mapper';
```

Comment:
```suggestion
import FormObjectMapper from '@components/form/form-object-mapper';
```
It will be more logical, and easier for research in code

Reply:
Meh, it's not related to the PR!

I fixed it anyway